operation/operation.c

```diff
@@ -165,6 +166,7 @@ void blink_until_sw_pressed(SW * sw_proceed, LED * led) {
         loop %= 10;
         sleep_ms(50);
     }
+    set_led_brightness(led, PWM_OFF);
```

Comment:
if button is pressed when the led is toggle on, the led will be on during calibration, line 169 is to be sure that the light is off after button is pressed